python/lsst/sconsUtils/utils.py

```diff
@@ -74,6 +74,23 @@ def libraryPathPassThrough():
 
 
 ##
+#  @brief Returns the full path to the Python executable as determined
```

Comment:
No, the intent was to be documenting the function `whichPython`. The two blank lines were there to appease the linter. Granted the variable should start with an underscore to indicate it's not a part of the public interface.